Examples/Physics_applications/pierce_diode/README.rst

```diff
@@ -0,0 +1,60 @@
+.. _examples-pierce-diode:
+
+Pierce Diode at the Child–Langmuir Limit
+===================
+
+This example shows how to simulate the physics of a 1D Pierce diode configuration operating
+at the Child–Langmuir limit using WarpX. In this setup, an electron beam is injected
+into a planar diode gap, consisting of by parallel conducting plates separated by the distance :math:`d` and powered by a voltage difference :math:`V`.
+For a suitable voltage, the field accelerates charged particles initially at rest (:math:`v = 0`) at the cathode towards the anode.
+The injected current density is chosen to match the space-charge-limited current predicted by the Child–Langmuir law :cite:t:`ex-Zhang2017`.
+
+Geometry
+---------
+
+The figure below schematically illustrates the problem geometry described above.
+
+.. figure:: https://gist.githubusercontent.com/oshapoval/aaafd8d131c3e1ed0fefe348bc8db28b/raw/92c4089e1b9eb23ae258f60c386e38e04f9499a2/geometry_pierce_diode.png
+   :alt: Two parallel conducting plates separated by the distance :math:`d` and powered by a voltage difference :math:`V` will.
+   :width: 80%
+   :align: center
+
+Сhild–Langmuir Limit
+---------
+
+In steady state, the emitted current is limited by the Child–Langmuir law,
+which defines the maximum current that can be transported across a planar diode for a given voltage and gap length :cite:t:`ex-Zhang2017`.
+From the Poisson solver, the current density can be expressed in terms of the voltage and the diode gap length, which in 1D writes as:
+
+.. math::
+   \phi(z)=V\Big(\frac{z}{d}\Big)^{4/3},
+   :label: child-langmuir-phi
+
+.. math::
+   J = \frac{4}{9} \varepsilon_0 \sqrt{\frac{2 |q|}{m}} \frac{|V|^{3/2}}{d^2}.
+   :label: child-langmuir-J
+
+Run
+---
+
+This example can be run with the WarpX executable using an input file: ``warpx.1d inputs_test_1d_pierce_diode``.
+For `MPI-parallel <https://www.mpi-forum.org>`__ runs, prefix these lines with ``mpiexec -n 4 ...`` or ``srun -n 4 ...``, depending on the system.
+
+.. literalinclude:: inputs_test_1d_pierce_diode
+   :language: ini
+   :caption: You can copy this file from ``Examples/Physics_applications/pierce_diode/inputs_test_1d_pierce_diode``.
+
+Visualize
+---------
+
+The figure below shows the results of the simulation (orange curves), which agrees well with the analytical Child–Langmuir law (black curves) (:eq:`child-langmuir-phi`, :eq:`child-langmuir-J`).
+
+.. figure:: https://gist.githubusercontent.com/oshapoval/aaafd8d131c3e1ed0fefe348bc8db28b/raw/fc76b371d323dbca4e1c43b45055405ff1fc6de4/Pierce_Diode.png
+   :alt: Results of the WarpX Pierce Diode simulation.
+   :width: 100%
+
```

Comment:
Would it be feasible to run the simulation with more macroparticles, so that the plot of `J` is less noisy?